qaframework-bdd-tests/src/test/resources/features/refapp-2.x/015-reports/reports.feature

```diff
@@ -14,4 +14,4 @@ Feature: Reports Management
     And user enters start date
     And user enters end date
     And user clicks on request report button
-    #Then user clicks on view report link
+    #Then user clicks on view report link
```

Comment:
is this change part of the ticket requirement?

Reply:
This change is part of the  requirement as its stated above.

Reply:
are the steps in report feature file used by Login steps definition?

Reply:
Please comfirm with the changes on master. this as you see it as a change was because of updating my branch to master. checkout this line https://github.com/openmrs/openmrs-contrib-qaframework/blob/master/qaframework-bdd-tests/src/test/resources/features/refapp-2.x/015-reports/reports.feature#L17

Reply:
> this as you see it as a change was because of updating my branch to master

@sherrif10 how do you update your branch?

Reply:
uhmm is this of help https://gist.github.com/whoisryosuke/36b3b41e738394170b9a7c230665e6b9

Reply:
When master changes are merged into a branch following the documentation you have shared, the changes are not reflected in the branch.